### ISSUE TITLE
VbMsgBoxStyle.vbCancelTryAgainContinue and VbMsgBoxResult.vbTryAgain/vbContinue

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#568]
**Is your feature request related to a problem? Please describe.**
Enhancing the MsgBox function by adding additional enum constants.

**Describe the solution you'd like**
Add vbCancelTryContinue (VbMsgBoxStyle)
```
Const MB_CANCELTRYCONTINUE As Long = 6
Const vbCancelTryAgainContinue = MB_CANCELTRYCONTINUE
```
and also add return values (VbMsgBoxResult)
```
Const IDTRYAGAIN As Long = 10, IDCONTINUE As Long = 11
Const vbTryAgain = IDTRYAGAIN
Const vbContinue = IDCONTINUE
```

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by WaynePhillipsEA** [comment]
Yes, this is a simple one.  Supported on Win 2000 / XP onwards.   My only question; shouldn't it be vbCancelTryAgainContinue?

**Comment by Kr00l** [comment]
> Yes, this is a simple one. Supported on Win 2000 / XP onwards. My only question; shouldn't it be vbCancelTryAgainContinue? I would've preferred vbCancelRetryContinue, but vbTryAgain <> vbRetry

Yes.vbCancelTryAgainContinue is better.

**Comment by WaynePhillipsEA** [comment]
This is now available in v0.13.37.  Thanks!
